alfred-telemetry-platform/src/main/java/eu/xenit/alfred/telemetry/binder/care4alf/LegacyClusteringMetrics.java

```diff
@@ -0,0 +1,41 @@
+package eu.xenit.alfred.telemetry.binder.care4alf;
+
+import io.micrometer.core.instrument.Gauge;
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.binder.MeterBinder;
+import javax.annotation.Nonnull;
+import javax.management.Attribute;
+import javax.management.AttributeList;
+import javax.management.MBeanServerConnection;
+import javax.management.ObjectName;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+
+/**
+ * Care4Alf only {@link MeterBinder} for legacy compatibility
+ */
+public class LegacyClusteringMetrics implements MeterBinder {
+    private static final Logger log = LoggerFactory.getLogger(LegacyClusteringMetrics.class);
+
+    @Autowired
+    private MBeanServerConnection alfrescoMBeanServer;
+
+    @Override
+    public void bindTo(@Nonnull MeterRegistry registry) {
+        Gauge.builder("alfresco.cluster.nodes", alfrescoMBeanServer, this::getNumClusterMembers)
+                .description("The amount of cluster nodes")
+                .register(registry);
+    }
+
+
+    protected int getNumClusterMembers(MBeanServerConnection alfrescoMBeanServer) {
+        try {
+            AttributeList attributes = alfrescoMBeanServer.getAttributes(new ObjectName("Alfresco:Name=Cluster,Tool=Admin"), new String[]{"NumClusterMembers"});
```

Comment:
This does not work on Alfresco Community (no jmx/mbeans support). Do we disable this somehow on alfresco-community ?

Reply:
Aha, `ClusteringMetricsBeanPostProcessor.java` - my bad :-)